### ISSUE TITLE
Add support for log analysis post upload

### ISSUE THREAD

**Issue by SamuelDudley** [SamuelDudley/APSyncCloud#1]
__Ping__ @davidbuzz
After a short chat with @fnoop the working plan is to install a minimal version of [maverick](https://github.com/fnoop/maverick) on the server with only the analysis tools enabled. See https://github.com/fnoop/maverick/issues/611 .

- Add maverick as a submodule to this repo
- Automate setup with scripts
- Write a couple of small scripts to expose / move the uploaded logs to the existing maverick log analysis workflow
- Provide a way for users to log into the server
- Expose the correct logs for the logged in user

**Comment by davidbuzz** [comment]
having looked at the maverick repo today, and done some deployment of things inside maveric using it's 'maverick' command-line,  it's repo seems to be very much focused on automating the assembly and build of mavericks own companion-computer images with all the tied-in dependancies that it entails.    It might be better to cherry-pick just the bit we need rather than submodule all of that.?

**Comment by davidbuzz** [comment]
...what we really want after all is just InfluxDB + Grafana + ( maybe ) Collectd  + fnoop/s tweaks that already do a great job of bringing this togetehr to analyse logs.

**Comment by fnoop** [comment]
@davidbuzz Not really, Maverick's focus is absolutely on automation of companion computers but the images are just a convenience for people to download and get started quickly.  Initially I didn't even provide the images but got some feedback that leaving people to bootstrap for 2 days on a raspberry isn't the most user-friendly approach :)
The config is very flexible and the architecture is completely modular, so you can simply switch everything off except for influxdb+grafana if that's just what you want.  I've raised an issue to track providing this config, as a starter:
https://github.com/fnoop/maverick/issues/611

If this isn't quite what is required then I'm happy to look at breaking the analysis module out into a separate github project.

**Comment by SamuelDudley** [comment]
If we can use the maverick-analysis components as per fnoop/maverick#611 at least the two projects would remain somewhat in sync and benefit from each other...
